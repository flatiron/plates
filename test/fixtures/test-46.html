<li id="zero"></li>
<li class="test"></li>
