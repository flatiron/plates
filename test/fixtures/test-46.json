{
  "zero": 0,
  "one": 1
}
