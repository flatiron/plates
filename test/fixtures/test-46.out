<li id="zero">0</li>
<li class="test">1</li>
